api/templates/design.go

```diff
@@ -868,7 +868,7 @@ var Input = Type("input", func() {
 	Attribute("componentId", String, "Component id for oauth kind inputs.", func() {
 		Example("keboola.ex-component")
 	})
-	Required("id", "name", "description", "type", "kind", "default")
+	Required("id", "name", "description", "type", "kind", "default", "componentId")
```

Comment:
Problém s prázdným `componentId` je zřejmě nějaký bug v API frameworku. Když není required, tak ten framework mu namísto stringu vygeneruje typ `*string` (pointer na string), což by samo o sobě asi nevadilo, hodnota se z něj dá vyzvednout přes ten `&` tady o kousek níž v `mapper.go`. Jenže ten framework to pak zřejmě kvůli nějakýmu tomu bugu nedokáže dostat do response. Tu hodnotu tam při debugu vidím až do poslední chvíle.

Tak bych to jako dočasný workaround dal required, čímž teda bude u všech inputů, i když prázdný.

Reply:
Pokud je to jen dočasný, tak asi ok.